R/fiveyeargwsum-render.R

```diff
@@ -104,14 +104,17 @@ createfiveyeargwsumPlot <- function(reportObject){
   # Add space to the top of the Y Axis
   plot_object <- RescaleYTop(plot_object)
 
+  #Add approval explanation label to the top of the plot
+  plot_object <- mtext(plot_object, text = "Displayed approval level(s) are from the source TS that statistics are derived from.", side=3, cex=0.6, line=0.1, adj=1, axes=FALSE)
+
   #Add Min/Max labels if we aren't plotting min and max
-  line <- 0.33
+  line <- 0.1
```

Comment:
I suppose I could just add a modifier based on the loop iteration each time rather than modifying the line value itself and then it might fit better into the styles.